src/SurfaceFluxes.jl

```diff
@@ -54,417 +57,752 @@ Surface flux conditions, returned from `surface_conditions`.
- - `transport` Type of transport (momentum or heat)
-
-"""
-function compute_physical_scale(
-    uf::UF.AbstractUniversalFunction{FT},
-    z_in,
-    z_0,
-    x_in,
-    x_s,
-    transport,
-    ::FVScheme,
-) where {FT}
-    von_karman_const::FT = CPSGS.von_karman_const(uf.param_set)
-    _π_group = FT(UF.π_group(uf, transport))
-    _π_group⁻¹ = (1 / _π_group)
-    R_z0 = 1 - z_0 / z_in
-    temp1 = log(z_in / z_0)
-    temp2 = -UF.Psi(uf, z_in / uf.L, transport)
-    temp3 = z_0 / z_in * UF.Psi(uf, z_0 / uf.L, transport)
-    temp4 = R_z0 * (UF.psi(uf, z_0 / uf.L, transport) - 1)
-    Σterms = temp1 + temp2 + temp3 + temp4
-    return _π_group⁻¹ * von_karman_const / Σterms * (x_in - x_s)
+  - param_set: Abstract Parameter Set containing physical, thermodynamic parameters.
+  - Cd: Momentum exchange coefficient
+  - sc: Container for surface conditions based on known combination
+        of the state vector, and {fluxes, friction velocity, exchange coefficients} for a given experiment
+  - scheme: Discretisation scheme (currently supports DG and FV)
+"""
+function momentum_fluxes(param_set, Cd, sc::AbstractSurfaceConditions, scheme)
+    ρ_sfc = TD.air_density(ts_sfc(sc))
+    ρτxz = -ρ_sfc * Cd * Δu1(sc) * windspeed(sc)
+    ρτyz = -ρ_sfc * Cd * Δu2(sc) * windspeed(sc)
+    return (ρτxz, ρτyz)
 end
 
-function compute_physical_scale(
-    uf::UF.AbstractUniversalFunction{FT},
-    z_in,
-    z_0,
-    x_in,
-    x_s,
-    transport,
-    ::DGScheme,
-) where {FT}
-    von_karman_const::FT = CPSGS.von_karman_const(uf.param_set)
-    _π_group = FT(UF.π_group(uf, transport))
-    _π_group⁻¹ = (1 / _π_group)
-    temp1 = log(z_in / z_0)
-    temp2 = -UF.psi(uf, z_in / uf.L, transport)
-    temp3 = UF.psi(uf, z_0 / uf.L, transport)
-    Σterms = temp1 + temp2 + temp3
-    return _π_group⁻¹ * von_karman_const / Σterms * (x_in - x_s)
+"""
+  sensible_heat_flux(param_set, Ch, sc, scheme)
+In cases where surface fluxes are known,
+return the known latent heat flux.
+## Arguments
+  - param_set: Abstract Parameter Set containing physical, thermodynamic parameters.
+  - Ch: Thermal exchange coefficient
+  - sc: Container for surface conditions based on known combination
+        of the state vector, and {fluxes, friction velocity, exchange coefficients} for a given experiment
+  - scheme: Discretisation scheme (currently supports DG and FV)
+"""
+function sensible_heat_flux(
+    param_set,
+    Ch,
+    sc::Union{ValuesOnly, Coefficients},
+    scheme,
+)
+    FT = eltype(sc.state_in.u)
+    grav::FT = CPP.grav(param_set)
+    cp_d::FT = CPP.cp_d(param_set)
+    R_d::FT = CPP.R_d(param_set)
+    T_0::FT = CPP.T_0(param_set)
+    cp_m = TD.cp_m(sc.state_in.ts)
+    ρ_sfc = TD.air_density(sc.state_sfc.ts)
+    T_in = TD.air_temperature(ts_in(sc))
+    T_sfc = TD.air_temperature(ts_sfc(sc))
+    ΔT = T_in - T_sfc
+    hd_sfc = cp_d * (T_sfc - T_0) + R_d * T_0
+    ΔΦ = grav * Δz(sc)
+    E = -ρ_sfc * Ch * windspeed(sc) * Δqt(sc)
```

Comment:
It might be better to make evaporation a separate function, so it can easily be reused.